docs/tutorial/force_calibration.rst

```diff
@@ -464,3 +464,102 @@ When working with small beads or at high laser powers, it is important to verify
+
+    ps = lk.calculate_power_spectrum(f2y.data, sample_rate=f2y.sample_rate, num_points_per_block=1, fit_range=(10, 23e3))
+    ps_blocked = ps.downsampled_by(200)
+
+Use a passive calibration model, with hydrodynamic correction turned on. Then perform a least-squares fit and plot the result::
+
+    model = lk.PassiveCalibrationModel(4.4, temperature=25.0, hydrodynamically_correct=True)
+    fit = lk.fit_power_spectrum(ps_blocked, model)
+
+    plt.figure()
+    fit.plot()
+    plt.title(
+        f'Skewed fit: $f_c$ = {fit.results["fc"].value:.1f}, $D$ = {fit.results["D"].value:.4f}, $f_d$ = {fit.results["f_diode"].value:.1f}'
+    )
+    plt.show()
+
+.. image:: figures/force_calibration/power_spectrum_noise_peak.png
+
+Notice how the tail of the model is skewed towards the peak, in order to reduce the least-squares error. The free parameters to fit the diode filter contribution are 'abused' to reduce the error between the model and the outlier.
+
+Now do a robust fit. We do this by specifying a loss function in the function `fit_power_spectrum()`. For least-squares fitting, the loss function is `'gaussian'`, which is the default if nothing is specified. However, if we specify `'lorentzian'`, a robust fitting routine will be used instead. Because `bias_correction` and robust fitting are mutually exclusive, we need to explicitly turn it off::
+
+    fit = lk.fit_power_spectrum(ps_blocked, model, bias_correction=False, loss_function="lorentzian")
+
+Now plot the robust fit::
+
+    plt.figure()
+    fit.plot()
+    plt.title(
+        f'Robust fit: $f_c$ = {fit.results["fc"].value:.1f}, $D$ = {fit.results["D"].value:.4f}, $f_d$ = {fit.results["f_diode"].value:.1f}'
+    )
+    plt.show()
+
+.. image:: figures/force_calibration/power_spectrum_noise_peak_robust.png
+
+Notice how the model now follows the power spectrum nearly perfectly. The value for `f_diode` has increased significantly, now that it is not abused to reduce the error induced by the outlier. 
+
+This example shows that a robust fitting method is less likely to choke on outliers in the power spectrum data. It is therefore a fair question why one would not use it all the time? First of all, robust fitting leads to a small bias in the fit results for which pylake has no correction. Least-squares fitting also leads to a bias, but this can be corrected with `bias_correction=True`. Secondly, for least-squares fitting, a large pool of literature is available to estimate the errors in the fit of the free parameters, which are implemented in the least-squares fitting routines that Pylake uses. These error estimates are not implemented for robust fitting, and as such, the fit results will show `nan` for the error estimates after a robust fit. However, as will be shown below, the robust fitting results may be used as a start to identify outliers automatically, in order to exclude these from a second, regular least-squares, fit. 
```

Comment:
```suggestion
This example shows that a robust fitting method is less likely to fail on outliers in the power spectrum data. It is therefore a fair question why one would not use it all the time? Robust fitting leads to a small bias in the fit results for which Pylake has no correction. Least-squares fitting also leads to a bias, but this bias is known (:cite:`norrelykke2010power`) and can be corrected with `bias_correction=True`. Secondly, for least-squares fitting, methods exist to estimate the expected standard errors in the estimates of the free parameters, which are implemented in the least-squares fitting routines that Pylake uses :cite:`press1990numerical`. These error estimates are not implemented for robust fitting, and as such, the fit results will show `nan` for the error estimates after a robust fit. However, as will be shown below, the robust fitting results may be used as a start to identify outliers automatically, in order to exclude these from a second, regular least-squares, fit. 
```

I would actually love to implement profile likelihood for these calibrations as well, since the standard errors obtained from these asymptotic methods can be deceptive when dealing with a pathological case.

Reply:
I agree, and honestly I'm not 100% convinced about the 'statistical backing' figure we get right now.